docs/source/verticapylab_gs.rst

```diff
@@ -0,0 +1,141 @@
+
+Install WSL 2
+~~~~~~~~~~~~~~
+
+1. Open the command prompt or Windows PowerShell as **Administrator**.
+
+2. Run the following command, which enables the required components, downloads the latest Linux kernel, sets WSL 2 as default, and installs the default Ubuntu distribution:
+	
+    .. code-block:: 
+        
+      $ wsl --install   
+
+
+    .. note:: The above command works only if WSL is not already installed. If the above command returns the WSL help text, run `wsl --list --online` to see a list of available Linux distributions, then run `wsl --install -d `*`DistroName`*  to install one of the available distributions. To uninstall WSL, see `unregister or uninstall a Linux distribution <https://docs.microsoft.com/en-us/windows/wsl/basic-commands#unregister-or-uninstall-a-linux-distribution>`_.
+
+3. Reboot your machine. The Linux distribution you installed above should start automatically. If it does not, you can start it manually from the Windows Start menu.
+
+4. The first time you launch a newly installed Linux distribution, a console window opens asking you to wait for files to decompress and be stored on your machine. All future launches should take less than a second.
+
+5. Set your username and password.
+6. Install the **make** package:
+
+    .. code-block::
+
+      $ sudo apt install make
+
+
+Docker Desktop
+---------------
+
+Docker Desktop is an out-of-the-box containerization software that includes a graphical user interface for managing your containers, applications, and images. Docker Desktop requires that WSL 2 is installed on your machine.
+
+Install and configure Docker Desktop
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Download and run the `Docker Desktop installer <https://docs.docker.com/desktop/windows/install/>`_, making sure to download a version of Docker Desktop using a Docker version 18.09 or higher. 
+2. Start Docker Desktop from the Windows Start menu.
+3. Click the gear icon on the top right and navigate to the **General** tab.
+4. Verify that the **Use WSL 2 based engine** option is turned on. If not, check it and click **Apply & Restart**.
+
+VerticaPyLab Quickstart
+========================
+
+The following steps import and launch VerticaPyLab:
+
+1. Start a Linux distribution.
+2. Clone the VerticaPyLab repository:
+
+    .. code-block::
+
+      $ git clone https://github.com/vertica/VerticaPyLab.git
+
+3. Navigate to the cloned directory:
+
+    .. code-block::
+        
+      $ cd VerticaPyLab
+
+4.  Start the VerticaPyLab service. There are two options, depending on whether you want to use an existing database or launch a demo database:
+	1. To create and connect to a demo database, run:
+	
```

Comment:
You can add a sentence here clarifying that this will launch both containers (Vertica + JupyterLab). Whereas the second option only launches one container.